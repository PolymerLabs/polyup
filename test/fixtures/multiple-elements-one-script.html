<link rel='import' href='../polymer/polymer.html'>

<polymer-element name='elem-1' attributes='one'>
</polymer-element>


<polymer-element name='elem-2' attributes='two'>
<script src='multiple-elements-one-script.js'></script>
</polymer-element>
