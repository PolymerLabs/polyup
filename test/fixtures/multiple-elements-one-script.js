Polymer({});

Polymer('elem-1', {})
