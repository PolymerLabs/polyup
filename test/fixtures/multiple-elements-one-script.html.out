<link rel="import" href="../polymer/polymer.html">

<dom-module id="elem-1">
</dom-module>


<dom-module id="elem-2">
  <script src="multiple-elements-one-script.js"></script>
</dom-module>
